webapp/components/admin_console/webhook_settings.jsx

```diff
@@ -124,7 +124,7 @@ export default class WebhookSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.service.overrideTitle'
-                            defaultMessage='Enable Overriding Usernames from Webhooks and Slash Commands: '
+                            defaultMessage='Enable integrations to override usernames: '
```

Comment:
Propose "Enable webhooks and slash commands to override usernames" // OAuth provider is being added to integrations and this setting won't apply.